src/Tribe/Customizer/Section.php

```diff
@@ -108,24 +108,6 @@ public function create_ghost_settings( $settings = array() ) {
 	private static $instances;
 
 	/**
-	 * Method to return the Private instance of the Class
-	 *
-	 * @since 4.0
-	 *
-	 * @access public
-	 * @return Tribe__Events__Pro__Customizer__Section
-	 */
-	public static function instance( $name ) {
```

Comment:
Removing this breaks how the sections Works on the Current Code. It uses this Slug to store data on the database option